scss/_patterns_modal.scss

```diff
@@ -51,6 +52,11 @@
     display: flex;
     justify-content: space-between;
     margin-bottom: $spv-inner--small;
+    padding-top: $spv-inner--large;
+    position: sticky;
+    top: 0;
+    z-index: 10;
+    background: $color-x-light;
```

Comment:
This may need some sorting as well. You can also run `dotrun test` locally to check.